graylog2-server/src/main/java/org/graylog2/rest/resources/system/lookup/LookupTableResource.java

```diff
@@ -173,9 +176,29 @@ public LookupTableResource(DBLookupTableService dbTableService,
         this.cacheSearchQueryParser = new SearchQueryParser(CacheDto.FIELD_TITLE, CACHE_SEARCH_FIELD_MAPPING);
     }
 
+    private void checkLookupTableId(String idOrName, LookupTableApi toUpdate) {
+        requireNonNull(idOrName, "idOrName parameter cannot be null");
+        if (!idOrName.equals(toUpdate.id()) || !idOrName.equals(toUpdate.name())) {
```

Comment:
There's still the edge case that the name changed and is used as the identifier for updating.
Our code doesn't do that, but I think we can live with it  for now.

If that's not the case, we can load the DTO first, and then check the old version of them.